torch/_numpy/_funcs.py

```diff
@@ -0,0 +1,73 @@
+import inspect
```

Comment:
We seem to have lost the README for torch/_numpy folder; having one would be pretty useful! Explain the overall structure, how it's tested, how to add new functions, etc.

Reply:
Yep, it's on my list. Will work on that today.